### PR TITLE
Validation form

### PR DESCRIPTION
**Pull request by elmar8287**
- Modify HTML code, add span for the error message
- Add in JS the validation
- Style the error message to red color

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,5 +6,201 @@ function closeNav() {
...
+const email = document.getElementById('user_email');
+const errorMessage = document.getElementById('emsg');
+form.addEventListener('submit', (x) => {
+  if (email.value.toLowerCase() !== email.value) {
+    x.preventDefault();
+    errorMessage.textContent = 'Use only lowercase letters for your e-mail!';
+    errorMessage.style.color = "red";
+    errorMessage.style.backgroundColor = "yellow";
+  }
+});
```
Comment (DeliceLydia) [inline index.js]: - [x] Error message has to fits the design(
font, colors, images, tests, margins)  please fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @elmar8287 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (elmar8287) [reply index.js]: Dear @DeliceLydia,

It is fixed, please check

MR

Elmar

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

Hi @elmar8287 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Dear @DeliceLydia! 🥇 

Many thanks for the review. I appreciate your comments 👍 

Many thanks and regards,

Elmar

**Pull request merged**